paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/RemoveOrphanFilesActionFactory.java

```diff
@@ -64,9 +100,11 @@ public void printHelp() {
                 "  remove_orphan_files \\\n"
                         + "--warehouse <warehouse_path> \\\n"
                         + "--database <database_name> \\\n"
-                        + "--table <table_name> \\\n"
+                        + "[--table <table_name>] \\\n"
+                        + "[--tables <table1,table2,...>] \\\n"
```

Comment:
The usage of --tables is confused. Is this a single parameter or a multi-parameter?